docker/linux/amazonlinux.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM amazonlinux:2
+
+RUN amazon-linux-extras enable python3.8
+
+RUN yum update -y
+RUN yum install -y git make python38 gcc10.x86_64 gcc10-c++.x86_64 binutils
+
+RUN git clone --branch v22.x --depth 1 https://github.com/nodejs/node.git
+ENV CC=gcc10-cc
+ENV CXX=gcc10-c++
+RUN cd node && ./configure
+RUN cd node && make -j4 -s > /dev/null
+RUN cd node && make install
+
+FROM amazonlinux:2
+
+COPY --from=0 /usr/local/bin/node /usr/local/bin/node
+RUN yum update -y
+RUN yum install -y gcc-c++ binutils make java-11-amazon-corretto.x86_64 patchelf tar
+RUN yum clean all
```

Comment:
Let's combine multiple `RUN` commands into one to reduce the number of layers in the docker image?

Reply:
21e5ca35b91e8db583efacbb4d1e4ec30b3b2fe8

Reply:
21e5ca35b91e8db583efacbb4d1e4ec30b3b2fe8

Reply:
I left the ones in the previous image because they provide useful checkpointing during a lengthy process like building Node.js